parts/common/components.json

```diff
@@ -1274,7 +1290,7 @@
               {
                 "k8sVersion": "1.32",
                 "renovateTag": "OCI_registry=https://mcr.microsoft.com, name=oss/binaries/kubernetes/kubernetes-node",
-                "latestVersion": "v1.32.1",
+                "latestVersion": "v1.32.2",
```

Comment:
This seems odd that we're updating latestVersion and not updateing previousLatestVersion - usually we'd change previousLatest to the previous latest rather than leave it the same.

Reply:
correct. The idea is `lastest` and `latest-1`